src/book.py

```diff
@@ -129,12 +131,14 @@ def clean(self, lemmatize=True, deromanize=False, lemma_pos='v', inplace=False):
         inplace (default False): replaces the text of the object with cleaned text
         """
         cleaned = self._text
+        '
```

Comment:
Delete this, it causes an error to be thrown so you can no longer import the module
```suggestion
```

Reply:
typo.